src/Form/Toggle/Toggle.component.tsx

```diff
@@ -164,6 +165,7 @@ export const Toggle = forwardRef(
             <HiddenInput
                 type="checkbox"
                 id={id}
+                onChange={onChange}
```

Comment:
onChange was being spread across both the `label` and the `input` causing the onChange handler to be called twice.

Reply:
👍